src/index.tsx

```diff
@@ -196,21 +216,23 @@ export default class ReactToPrint extends React.Component<IReactToPrintProps> {
             const onBeforeGetContentOutput = onBeforeGetContent();
             if (onBeforeGetContentOutput && typeof onBeforeGetContentOutput.then === "function") {
                 onBeforeGetContentOutput
-                    .then(this.handlePrint)
+                    .then(() => this.handlePrint(content))
                     .catch((error: Error) => {
                         if (onPrintError) {
                             onPrintError("onBeforeGetContent", error);
                         }
                     });
             } else {
-                this.handlePrint();
+                this.handlePrint(content);
             }
         } else {
-            this.handlePrint();
+            this.handlePrint(content);
         }
+
+        return Boolean(event), true;
```

Comment:
What is the purpose of this return statement? Can it be removed? It seems to always return `true`, even if `Boolean(event) === false` (honestly when I first saw this I didn't even think it was valid JS, learn something new every day!

Reply:
There's a need for this line else `event` becomes and _unused variable_ and the `tsc` starts throwing errors

Reply:
Does changing the parameter name to `_event` solve that?

Reply:
However, i have resolved it using an `eslint` comment